coordinator/src/services/translation/prescription/index.ts

```diff
@@ -184,9 +180,7 @@ function convertPrescriptionPertinentInformation2(
   const pertinentInformation2 = []
 
   for (let i = 0; i < fhirMedicationRequests.length; i++) {
-    const patientInfoText = isFirstRequestAndCommunicationRequestPresent(i, fhirCommunicationRequests)
-      ? extractText(fhirCommunicationRequests)
-      : []
+    const patientInfoText = i === 0 ? extractText(fhirCommunicationRequests) : []
```

Comment:
A comment to explain that we only want to set the patient info text for the first item might be useful, or alternatively (to 'self document' the code, create a small 'isFirstMedicationRequest` function.